internal/output/plain.go

```diff
@@ -324,3 +330,30 @@ func trimValue(value string, size int) string {
 	}
 	return value
 }
+
+func shiftColsVal(opts []string) int {
+	for _, opt := range opts {
+		a := strings.TrimPrefix(opt, string(ShiftColsPrefix))
+		if len(a) < len(opt) {
```

Comment:
This conditional seems error prone and pointless. Granted it would look ridiculous if we passed an int that's longer in string size than the prefix, but that doesn't take away that this is unnecessary. Just drop the conditional here.

Reply:
This ensures that a potential value was found. The length of "a" would be the remaining string-integer except when no prefix is trimmed (i.e. no shiftCols option was set).

Reply:
In order to improve readability, the var name has been updated and a comment was added to the conditional line.